CalculatorWithTests/Calculator.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+
+
+
+namespace CalculatorWithTests
+{
+    public class Calculator
+    {
+       public int Add(int number1, int number2)
+        {
+            return (number1 + number2);
+        }
+
+
```

Comment:
This empty line should be removed, leaving only 1 empty separating these methods